src/test_suite/interfaces.ts

```diff
@@ -38,3 +38,10 @@ export interface TestLineItem {
 export interface TestOrder {
     readonly cart: TestLineItem[];
 }
+
+export enum TextType {
+    None = 'DEFAULT',
```

Comment:
I wonder if we'd be better off using a numeric enum without initializers here. The question to ask is, what is the benefit of the string value in this scenario. Is the value ever serialized out or printed?

Reply:
Also, I don't think we need the None level.

Reply:
the value is printed out when they provide the -t argument, but we don't have to?